src/hotspot/share/gc/shenandoah/shenandoahControlThread.cpp

```diff
@@ -448,7 +448,7 @@ void ShenandoahControlThread::process_phase_timings(const ShenandoahHeap* heap)
 //
 void ShenandoahControlThread::service_concurrent_normal_cycle(
   const ShenandoahHeap* heap, const GenerationMode generation, GCCause::Cause cause) {
-
+  GCIdMark gc_id_mark;
```

Comment:
Why does the GCIdMark need to move around?

Reply:
I pulled up the gcid mark because every old generation collection is preceded by a "bootstrap" young collection. Logically, this bootstrap phase belongs to the old collection so it should have the same GC id as the subsequent old marking phase.